app/views/view_submissions.scala.html

```diff
@@ -14,55 +14,99 @@
  * limitations under the License.
  *@
 
+@import components.gds.{gds_main_template, pageTitle}
 @import java.time.format.DateTimeFormatter
-
 @import models.notifications.NotificationFrontendModel
 @import models.submissions.Submission
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.components.BackButton
 @import views.Title
 
-@this(main_template: views.html.templates.main_template, formatter: DateTimeFormatter)
+@this(
+  govukLayout: gds_main_template,
+  pageTitle: pageTitle,
+  table: GovukTable,
+  formatter: DateTimeFormatter
+)
 
 @(submissions: Seq[(Submission, Seq[NotificationFrontendModel])])(implicit request: Request[_], messages: Messages)
 
+@ucrCellContent(submission: Submission, notifications: Seq[NotificationFrontendModel]) = {
+  @if(notifications.isEmpty){
+    @components.elements_in_separate_lines(submission.ucrBlocks.map(_.ucr))
+  } else {
+    <a href="@routes.ViewNotificationsController.listOfNotifications(submission.conversationId)">
```

Comment:
can we put a also in separate partial - same as `elements_in_separate_lines` ?
This could looks like:
```
components.link(routes.ViewNotificationsController.listOfNotifications(submission.conversationId)){
  @components.elements_in_separate_lines(submission.ucrBlocks.map(_.ucr))
}
```
```

Reply:
👍